Cargo.toml

```diff
@@ -1,21 +1,17 @@
 [package]
 name = "poseidon252"
-version = "0.1.0"
+version = "0.2.0"
 authors = [
   "zer0 <matteo@dusk.network>"
 ]
 edition = "2018"
 
 [dependencies]
-curve25519-dalek = { version = "2", features = ["serde", "avx2_backend"] }
+dusk-bls12_381 = "0.1"
 merlin = "2"
 lazy_static = "1.3.0"
-hades252 = { git = "https://github.com/dusk-network/hades252" }
-
-[dependencies.bulletproofs]
-git = "https://github.com/dusk-network/bulletproofs"
-branch = "develop"
-features = ["yoloproofs"]
+hades252 = { git = "https://github.com/dusk-network/hades252", branch = "master" }
```

Comment:
Which version of Hades are we targeting?

Reply:
I'm picking "master" since there's no tag or version for the newes't implementation which is using PLONK as constraint system & proof system.

I think that you were going to add the tag once the history was re-written. But I checked and it hasn't been created.